docs/develop/convert-javascript-to-typescript.md

```diff
@@ -127,7 +114,7 @@ For your reference, the following code snippet shows the contents of the **Home.
             var element = document.querySelector('.MessageBanner');
             messageBanner = new components.MessageBanner(element);
             messageBanner.hideBanner();
-            
+
             // If not using Excel 2016, use fallback logic.
```

Comment:
Can you clarify the Excel versions here? Is this condition applicable to Excel 2013 or Excel 2019?

```suggestion
            // If not using Excel 2016 or later, use fallback logic.
```

Reply:
Good call. I flipped it to explicitly call out the usage of Excel 2013. I think it's more clear to point to 2013 as the outlier.